webpack.config.babel.js

```diff
@@ -23,6 +23,7 @@ export default {
   externals: {
     react: 'React',
     'react-dom': 'ReactDOM',
+    'styled-components': 'styled',
```

Comment:
shouldn't this be `'styled-components': 'styled-components'?

Also, I noticed the following addition and thought I'll share it for awareness:
https://github.com/styled-components/styled-components/issues/1770#issuecomment-394101268

Reply:
The correct variable name is `styled`, as in `window.styled`, which aligns with the default export name.

I think that comment is just if using `npm link`.